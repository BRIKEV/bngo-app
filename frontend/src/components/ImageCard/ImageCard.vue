<template>
  <div class="imageCard"></div>
</template>

<script>
export default {
  name: 'ImageCard',
};
</script>
