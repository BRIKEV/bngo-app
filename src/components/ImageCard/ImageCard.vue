<template>
  <div class="imageCard">
    <img class="image" :src="imageUrl" alt="" />
  </div>
</template>

<script>
import VueTypes from 'vue-types';

export default {
  name: 'ImageCard',
  props: {
    imageUrl: VueTypes.string.isRequired,
  },
};
</script>
<style lang="scss" scoped>
$boxShadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
.imageCard {
  height: 300px;
  width: 300px;
}
.image {
  height: 100%;
  width: 100%;
  border-radius: 50%;
  box-shadow: $boxShadow;
}
</style>
