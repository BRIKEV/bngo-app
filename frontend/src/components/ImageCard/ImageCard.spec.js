import { mount } from '@vue/test-utils';
import ImageCard from './ImageCard.vue';

describe('ImageCard component', () => {
  const propsData = {
    imageUrl: 'urlOfimgae',
  };
  const defaultConfig = {
    propsData,
  };
  it('Should be render correctly', () => {
    const wrapper = mount(ImageCard, defaultConfig);
    expect(wrapper.element).toMatchSnapshot();
  });
});
