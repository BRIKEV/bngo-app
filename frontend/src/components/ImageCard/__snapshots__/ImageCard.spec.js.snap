// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageCard component Should be render correctly 1`] = `
<div
  class="imageCard"
>
  <img
    alt=""
    class="image"
    src="urlOfimgae"
  />
</div>
`;
